### PR TITLE
Docv4 and Data types

### PR DESCRIPTION
**Pull request by Vishu26**

### CODE REVIEW

File: bspysmg/model/training.py [hunk 1]
```diff
@@ -121,17 +139,17 @@ def generate_surrogate_model(
...
-    optimizer,
-    epochs,
-    amplification,
-    start_epoch=0,
-    save_dir=None,
-    early_stopping=True,
-):
+    model : torch.nn.Module,
+    info_dict : dict,
+    dataloaders : list,
```
Comment (hcruiz) [inline bspysmg/model/training.py:144]: what type have the elements of this list?

File: bspysmg/model/training.py [hunk 2]
```diff
@@ -362,7 +387,25 @@ def postprocess(dataloader, model, criterion, amplification, results_dir,
     return torch.sqrt(running_loss)
 
 
-def to_device(inputs, targets):
+def to_device(inputs : torch.Tensor,
```
Comment (hcruiz) [inline bspysmg/model/training.py:390]: Why not use just a single argument? It is irrelevant if it is inputs and targets?

**Review by hcruiz — CHANGES_REQUESTED** [review]
please specify the type in the list for dataloaders

File: bspysmg/model/training.py (hunk 1 above)
Comment (Vishu26) [reply bspysmg/model/training.py]: Made the necessary change - List[torch.utils.data.DataLoader]

File: bspysmg/model/training.py (hunk 2 above)
Comment (Vishu26) [reply bspysmg/model/training.py]: Made necessary changes.

**Pull request merged**